config/api.yml

```diff
@@ -121,6 +121,10 @@
       :delete:
       - :name: delete
         :identifier: alert_definition_profile_delete
+    :alert_definitions_subcollection_actions:
+      :post:
+      - :name: assign
```

Comment:
Are there no RBAC identifiers for there actions? This seems surprising, since you need at least some privileges just to read an alert definition.

Reply:
I don't see any in `miq_product_features.yml`, so I don't think there are. Should I use the closest thing I can find, `alert_definition_edit`, or keep it without an identifier?

Reply:
It sounds like you may need to create new ones if they don't exist.

Reply:
Oh, I was wrong, it does exist, there's `alert_profile_assign`. I'll modify the commit accordingly

Reply:
`alert_profile_assign` appears to be unrelated. The UI uses `alert_definition_profile_edit` for assignment/unassignment, so I changed my code to use that instead.